packages/components/board/src/board.js

```diff
@@ -0,0 +1,49 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import css from './board.css';
+
+customElementDefineHelper(
+	'ts-board',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				dir: { type: String, reflect: true },
+				title: { type: String, attribute: 'data-title' }
+			};
+		}
+
+		constructor() {
+			super();
+			this.title = '';
+		}
+
+		get direction() {
+			return this.dir ? this.dir : this.bodyDir;
+		}
+
+		get header() {
+			if (!this.title) {
+				return '';
+			}
+			return html`
+				<header class="board-header">
+					<div class="board-title">${this.title}</div>
+				</header>
+			`;
+		}
+
+		render() {
+			return html`
+				<div dir="${this.direction}" class="board-container">
+					${this.header}
+					<main class="board-main">
+						<slot name="main"></slot>
```

Comment:
What if we don't put a name for it?

Reply:
I will add more slots. such as `tab`, `Button`.
Let's keep it right now.